packages/webview_flutter/webview_flutter_wkwebview/pigeons/web_kit.dart

```diff
@@ -101,19 +101,19 @@ class WKAudiovisualMediaTypeEnumData {
 /// Mirror of WKWebsiteDataTypes.
 ///
 /// See https://developer.apple.com/documentation/webkit/wkwebsitedatarecord/data_store_record_types?language=objc.
-enum WKWebsiteDataTypesEnum {
+enum WKWebsiteDataTypeEnum {
   cookies,
   memoryCache,
   diskCache,
   offlineWebApplicationCache,
-  localStroage,
+  localStorage,
   sessionStorage,
-  sqlDatabases,
+  webSQLDatabases,
   indexedDBDatabases,
 }
 
-class WKWebsiteDataTypesEnumData {
-  late WKWebsiteDataTypesEnum? value;
+class WKWebsiteDataTypeEnumData {
```

Comment:
Can we eliminate this wrapper? I forget if direct used of null enums are part of what we fixed, or if they are still broken.

Reply:
This is still required due to: https://github.com/flutter/flutter/issues/87307. I updated all the wrappers with a `TODO`. However, it looks like https://github.com/flutter/flutter/issues/100594 was fixed. So, I updated all the data classes.